specification/sql/resource-manager/Microsoft.Sql/preview/2023-05-01-preview/JobAgents.json

```diff
@@ -362,6 +426,24 @@
           }
         }
       }
+    },
+    "JobAgentUserAssignedIdentity": {
```

Comment:
>JobAgentUserAssignedIdentity [](http://example.com/codeflow?start=5&length=28)

Please replace with references to appropriate definitions from [[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json)](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json`) or reply to explain any problems with that.

Reply:
From other conversation: 
We would like to keep this consistent with other Microsoft.Sql APIs such as: https://github.com/Azure/azure-rest-api-specs/blob/def187e2e78d7173d8fdd7f77740dd9719e1dfbf/specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/Databases.json#L875C7-L875C7